frontend/services/viewer/viewer.ts

```diff
@@ -6,11 +6,16 @@ const MODE = {
 	PIN: 'PIN'
 };
 
+export const INITIAL_HELICOPTER_SPEED = 1;
```

Comment:
Is there a reason to define these constants in frontend/services/viewer/viewer.ts instead of frontend/constants/viewer.ts?

Reply:
Good point! It should be moved to viewer constants.
@dleszcz

Reply:
Fixed